src/main/java/DukeException.java

```diff
@@ -0,0 +1,2 @@
+public class DukeException extends Exception {
```

Comment:
Consider making a constructor with an error message, so you can preserve different errors and print their messages when making and returning DukeExceptions!